### PR TITLE
#3124: fix disappearing traces

### LINKED ISSUE #3124: Trace icon never appears and/or trace disappears when other extension runs
Link to Slack discussion with Looms: https://pixiebrix.slack.com/archives/C02DKNNB7T9/p1649299526953759

- First loom: it looks like when the panel loads (which is another extension), it clears out the trace from the extension that he ran that Mike's editing in the page editor
- Second loom: it's skipping one of the bricks

I don't think these will be fixed by: https://github.com/pixiebrix/pixiebrix-extension/pull/3117?

### PR DESCRIPTION
**Pull request by BALEHOK**
Fixes #3124

### CODE REVIEW

File: src/pageEditor/pageEditor.tsx [hunk 1]
```diff
@@ -30,10 +30,13 @@ import Panel from "@/pageEditor/Panel";
 import { watchNavigation } from "@/pageEditor/protocol";
 import initGoogle from "@/contrib/google/initGoogle";
 import { initToaster } from "@/utils/notify";
+import { clearDebugLogs, traces } from "@/background/messenger/api";
 
 registerMessenger();
 initGoogle();
 watchNavigation();
 initToaster();
+traces.clearAll();
```
Comment (twschiller) [inline src/pageEditor/pageEditor.tsx:39]: I think this approach is OK for this PR. As discussed over Slack/Zoom, let's also clear on extension run

**Review by twschiller — APPROVED** [review]

**Comment by BALEHOK** [comment]
Output of a skipped brick
image

File: src/extensionPoints/contextMenu.ts [hunk 2]
```diff
@@ -298,7 +301,7 @@ export abstract class ContextMenuExtensionPoint extends ExtensionPoint<ContextMe
           optionsArgs: extension.optionsArgs,
         };
 
-        await reducePipeline(actionConfig, initialValues, {
+        await reduceExtensionPipeline(actionConfig, initialValues, {
```
Comment (twschiller) [inline src/extensionPoints/contextMenu.ts:304]: 👍

File: src/extensionPoints/menuItemExtension.ts [hunk 3]
```diff
@@ -544,7 +548,7 @@ export abstract class MenuItemExtensionPoint extends ExtensionPoint<MenuItemExte
           root: document,
         };
 
-        await reducePipeline(actionConfig, initialValues, {
+        await reduceExtensionPipeline(actionConfig, initialValues, {
```
Comment (twschiller) [inline src/extensionPoints/menuItemExtension.ts:551]: 👍

File: src/extensionPoints/panelExtension.ts [hunk 4]
```diff
@@ -380,7 +383,7 @@ export abstract class PanelExtensionPoint extends ExtensionPoint<PanelConfig> {
           root: document,
         };
 
-        const rendererPromise = reducePipeline(body, initialValues, {
+        const rendererPromise = reduceExtensionPipeline(body, initialValues, {
```
Comment (twschiller) [inline src/extensionPoints/panelExtension.ts:386]: 👍

File: src/extensionPoints/quickBarExtension.tsx [hunk 5]
```diff
@@ -224,7 +227,7 @@ export abstract class QuickBarExtensionPoint extends ExtensionPoint<QuickBarConf
             optionsArgs: extension.optionsArgs,
           };
 
-          await reducePipeline(actionConfig, initialValues, {
+          await reduceExtensionPipeline(actionConfig, initialValues, {
```
Comment (twschiller) [inline src/extensionPoints/quickBarExtension.tsx:230]: 👍

**Review by twschiller — APPROVED** [review]

**Pull request merged**